csharp/ql/test/library-tests/regressions/Program.cs

```diff
@@ -147,4 +147,78 @@ partial class C1<T> where T: DynamicType
 {
 }
 
+namespace NoPia
```

Comment:
How about a smaller example that still exhibits the issue?

Reply:
Fair enough. Managed to reduce it further.